hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -671,6 +687,8 @@
       name: chip_clk_div
       desc: '''Verify clk division logic is working correctly.
 
+            This can be easily be verified if we implement clock cycle counters.
+            The IP level checks the divided clocks on each parent clock cycle.
```

Comment:
I think the integration check is to prove that the blocks that are expected to receive div4 clocks are indeed receiving div4 clocks. If we go with our 'IP-assertion-checks-in-bind-reused-at-chip' strategy, then all we need in addition is connectivity checks.

Reply:
Added mention of the required connectivity tests.